_data/finance.yml

```diff
@@ -49,14 +54,16 @@ websites:
       doc: https://support.gusto.com/hc/en-us/articles/210838787-Two-Step-Authentication
 
     - name: HelloWallet
-      url: https://www.hellowallet.com
+      url: http://www.hellowallet.com
       twitter: HelloWallet
+      facebook: hellowallet
       img: hellowallet.png
       tfa: No
 
     - name: InEx Finance
       url: https://www.inexfinance.com/
       twitter: InexFinance
+      facebook: inex.finance
```

Comment:
I'm not sure about this one. I can't find a link to it from their website. I only end up at [fentury](https://www.facebook.com/fentury).